back-end/api/quickstart/views.py

```diff
@@ -38,6 +38,24 @@ class AuthorViewSet(viewsets.ModelViewSet):
     lookup_field = 'id'
 
 
+class AuthUserViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows to fetch an author by the author's username (NOT displayName).
+    """
+    queryset = Author.objects.all()
+    serializer_class = AuthorSerializer
+    lookup_field = 'id'
```

Comment:
Does it need lookup_field anymore since you overrided the retrieve method?

Reply:
Good point. Removed it